main/main.c

```diff
@@ -1926,6 +1924,8 @@ void php_request_shutdown(void *dummy)
 		php_output_end_all();
 	} zend_end_try();
 
+	php_deactivate_ticks();
```

Comment:
I think the same issue can occur with buffered output handlers. Does this need to be moved down further?

Reply:
This is already below `php_output_end_all` and I don't think `php_output_deactivate` can call user code?

Reply:
Ah, you're probably right! Can't verify as I'm ok mobile.

Reply:
I think it doesn't call user code, at least not on purpose. Destructors are already called, so destroying the user output handler will not call any user code. The only way for `php_output_deactivate()` to execute user code would be if it invoked an error handler. But this would be unsafe after `zend_deactivate_modules()`, so it probably doesn't.

Reply:
Indeed. There's even a test. I just didn't look at it well enough.